### ISSUE TITLE
Integration test for LTR on MS MARCO

### ISSUE THREAD

**Issue by lintool** [castorini/pyserini#600]
@yuki617 @stephaniewhoo We should have an integration test for MS MARCO LTR, at least this part:

```
python scripts/ltr_msmarco-passage/predict_passage.py \
  --rank_list_path runs/run.msmarco-passage.bm25tuned.txt \
  --rank_list_format tsv \
  --ltr_model_path runs/msmarco-passage-ltr-mrr-v1 \
  --ltr_output_path runs/run.ltr.msmarco-passage.tsv 
```

Should be easy to set up... so let's do it now so that a task like #599 will be easier to do...

**Comment by stephaniewhoo** [comment]
commits to PR [584](https://github.com/castorini/pyserini/pull/584)
@yuki617 pls review

**Comment by lintool** [comment]
Closed by #584
